docs/examples/retrievers/composable_retrievers.ipynb

```diff
@@ -0,0 +1,399 @@
+{
```

Comment:
<p>it might be good to view a "verbose" output so that users can get a better sense of sense of the execution trace (and/or show our observability integration) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10019/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Not so easy to add verbose to the base retriever class. But I can highlight arize with this

Reply:
:sigh: adding verbose to the interface becasue arize doesn't want to work